napari_animation/_qt/animation_widget.py

```diff
@@ -40,37 +40,42 @@ def __init__(self, viewer: Viewer, parent=None):
         self.viewer = viewer
         self.animation = Animation(self.viewer)
 
-        # Initialise UI
-        self._init_ui()
+        # Initialise User Interface
+        self.keyframesListControlWidget = KeyFrameListControlWidget(
+            animation=self.animation, parent=self
+        )
+        self.keyframesListWidget = KeyFramesListWidget(
+            self.animation.key_frames, parent=self
+        )
+        self.frameWidget = FrameWidget(parent=self)
+        self.saveButton = QPushButton("Save Animation", parent=self)
+        self.animationsliderWidget = AnimationSliderWidget(
+            self.animation, orientation=Qt.Horizontal, parent=self
+        )
+
+        # Create layout
+        self.setLayout(QVBoxLayout())
+        self.layout().addWidget(self.keyframesListControlWidget)
+        self.layout().addWidget(self.keyframesListWidget)
+        self.layout().addWidget(self.frameWidget)
+        self.layout().addWidget(self.saveButton)
+        self.layout().addWidget(self.animationsliderWidget)
 
         # establish key bindings and callbacks
         self._add_keybind_callbacks()
         self._add_callbacks()
 
-    def _init_ui(self):
-        """Initialise user interface"""
-        self._layout = QVBoxLayout()
-        self.setLayout(self._layout)
-
-        self._init_keyframes_list_control_widget()
-        self._init_keyframes_list_widget()
-        self._init_frame_widget()
-        self._init_save_button()
-        self._init_animation_slider_widget()
-
     def _add_keybind_callbacks(self):
         """Bind keys"""
-
-        self.animation.viewer.bind_key(
-            "Alt-f", self._capture_keyframe_callback
-        )
-        self.animation.viewer.bind_key(
-            "Alt-r", self._replace_keyframe_callback
-        )
-        self.animation.viewer.bind_key("Alt-d", self._delete_keyframe_callback)
-
-        self.animation.viewer.bind_key("Alt-a", self._key_adv_frame)
-        self.animation.viewer.bind_key("Alt-b", self._key_back_frame)
+        self._keybindings = [
```

Comment:
hopefully one day we'll be able to use the new action manager stuff in napari here!!